directory.json

```diff
@@ -182,5 +182,6 @@
   "UN-Hack-Bar":"https://keinanschluss.un-hack-bar.de/spaceapi.json",
   "Zeus WPI":"https://mattermore.zeus.gent/spaceapi.json",
   "z-Labor Zwickau":"http://api.service.z-labor.space/spaceapi.json",
-  "SchonungsLos":"https://www.hackerspace-sw.de/spaceapi.json"
+  "SchonungsLos":"https://www.hackerspace-sw.de/spaceapi.json",
+  "Liege Hackerspace":"https://liegehacker.space/spaceapi/space.json"
```

Comment:
Hmm it looks like the entries are partially alphabetically sorted (with capital letters first as it seems). Should we try to keep it sorted?